nano/core_test/active_transactions.cpp

```diff
@@ -707,60 +711,68 @@ TEST (active_transactions, republish_winner)
 
 TEST (active_transactions, fork_filter_cleanup)
 {
-	nano::system system;
-	nano::node_config node_config (nano::get_available_port (), system.logging);
+	nano::system system{};
+
+	nano::node_config node_config{ nano::get_available_port (), system.logging };
 	node_config.frontiers_confirmation = nano::frontiers_confirmation_mode::disabled;
-	auto & node1 (*system.add_node (node_config));
 
-	nano::keypair key;
-	nano::state_block_builder builder;
+	auto & node1 = *system.add_node (node_config);
+	nano::keypair key{};
+	nano::send_block_builder builder{};
```

Comment:
This also reverts to legacy blocks.

Reply:
perhaps it should be called `legacy_send_block_builder` to prevent others from making that mistake

Reply:
Yeah it is a bit unfortunate -- I actually thought we're better off using send_, open_ etc builders since they're more specific, unbeknownst they refer to the legacy block types. Addressed, renaming might be a good idea too, any input on this @clemahieu ?